code/src/index.css

```diff
@@ -5,9 +5,130 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
+    
 }
+
+
+
+/*PopularList*/
+.all-movies {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  flex-wrap: wrap;
+  width: 100%;
+  box-sizing: border-box;
+  
+}
+
+.movie-container {
+  word-wrap: break-word;
+  text-align: center;
+  position: relative;
+}
+
+a.movie {
+  display: flex;
+
+}
+
+.all-movies .movie-info {
+  position: absolute;
+  height: 100%;
+  color: #f9f9f9;
+  text-align: center;
+  background: rgba(0,0,0,0.75);
+  display: none;
+  margin-top: auto;
+  margin-bottom: 0;
+}
+
+.all-movies a:hover .movie-info {
+  display: none;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: flex-end;
+  max-width: 100%;
+}
+
+
+
+.all-movie-posters {
+  width: 100%;
+  height: 100%;
+}
+
+/*ShowMovie*/
+.back-link {
```

Comment:
If you want to style the button more, this is how we styled our button. We used flexbox to position it.  :)

 .backButton {
    background-color: transparent;
    color: white;
    cursor: pointer;
    font-weight: 700;
    border: 5px white;
    padding: 8px 15px;
    opacity: 0.7;
    display: flex;
    flex-direction: row;
    gap: 10px;
  }